docs/BasicCommunication/OnEventChanged/index.md

```diff
@@ -18,20 +18,18 @@ SDL uses `OnEventChanged` notification to appropriately manage the hmiLevel and
 !!! MUST
 1. Send notification with appropriate parameter value when active call on HMI has been started/ended.
 
-2. Send [SDL.OnAppDeactivated](../../sdl/onappdeactivated) to the active app when the phone call is started.
-
-3. Resume the applications to their original state prior to the phone call event in the HMI when the event ends (see note below).
+2. Resume the applications to their original audio streaming state prior to the phone call event in the HMI when the event ends (see note below).
 !!!
 
 !!! NOTE
-SDL does not send BC.ActivateApp or BC.OnResumeAudioSource to HMI after the phone call is ended.
+If the HMI wants to leave the app screen when the PHONE_CALL event occurs, the app's HMI state can be changed accordingly using the `BC.OnAppDeactivated(appID)` and `BC.OnAppActivated(appID)` notifications.
 !!!
 
 Upon receiving `OnEventChanged(PHONE_CALL)`, SDL will:
 
 |isActive|Result|
 |:-------|:-----|
-|true|Change the HMI state of all media applications currently either in FULL or LIMITED to (BACKGROUND, NOT_AUDIBLE) and all navi/projection apps to (LIMITED, NOT_AUDIBLE, STREAMABLE)|
+|true|Keep the HMI state of all applications but change audible state of all applications to NOT_AUDIBLE|
```

Comment:
```suggestion
|true|Move all apps with audioStreamingState AUDIBLE to NOT_AUDIBLE|
```
I still think this should be modified to be consistent with the rest of the document (see `EMERGENCY_EVENT` section). I don't think that the wording in the proposal needs to be followed strictly in this case, it seems to be more of an example.

Reply:
@jacobkeeler -san,
Thank you for your comment.
We understood.
We made a commit your suggestion.
Thank you.